src/devana/code_generation/printers/default/variableprinter.py

```diff
@@ -28,6 +28,8 @@ def print(self, source: Variable, config: Optional[PrinterConfiguration] = None,
                 mods += "restrict "
             elif source.type.modification.is_constexpr:
                 mods += "constexpr "
+            elif source.type.modification.is_constinit:
+                mods += "constinit"
```

Comment:
Please write test of this printer. You have bug here - no space after keyword (please look other keywords in printer).